app/script/conversation/ConversationRepository.js

```diff
@@ -2099,16 +2100,16 @@ z.conversation.ConversationRepository = class ConversationRepository {
   /**
    * Send text message with link preview in specified conversation.
    *
-   * @param {string} textMessage - Plain text message
    * @param {Conversation} conversationEntity - Conversation that should receive the message
+   * @param {string} textMessage - Plain text message
    * @param {Array<z.message.MentionEntity>} [mentionEntities] - Mentions part of the message
    * @returns {Promise} Resolves after sending the message
    */
-  sendTextWithLinkPreview(textMessage, conversationEntity, mentionEntities) {
-    return this.sendText(textMessage, conversationEntity, mentionEntities)
+  sendTextWithLinkPreview(conversationEntity, textMessage, mentionEntities) {
+    return this.sendText(conversationEntity, textMessage, mentionEntities)
       .then(genericMessage => {
         if (z.util.Environment.desktop) {
-          return this.sendLinkPreview(textMessage, conversationEntity, genericMessage);
+          return this.sendLinkPreview(conversationEntity, textMessage, genericMessage);
```

Comment:
This was causing the actual bug. Since https://github.com/wireapp/wire-webapp/pull/4711/files#diff-2fd47a6e7a8f0ed5ea10c299b3145752R1919 already expected the `conversationEntity` as first param.